app/models/port.rb

```diff
@@ -59,6 +59,54 @@ def connect_to_port(other_port, cable_name, cable_color, vlans, special_case = n
     end
   end
 
+  def self.to_txt(frames)
+    txt = ""
```

Comment:
```suggestion
    txt = []
```

To make this compliant with Rails 6 upgrade